lib/automatic/api_client/trips.rb

```diff
@@ -0,0 +1,60 @@
+module Automatic
+  class ApiClient
+    def trips
+      Trips.new(self)
+    end
+
+    class Trips
```

Comment:
Essentially yeah - this was the initial way I addressed the issue that cropped up in https://github.com/lumoslabs/automatic-client/pull/2 before trying out the `method_missing` stuff. This took the class-level methods in `Models::Trips` and created duplicated instance-level methods but would need some pretty major DRYing out to be usable across the other model classes and methods